### ISSUE TITLE
Issue in make smartie-sv

### ISSUE THREAD

**Issue by AishaYousaf** [zeeev/smartie-sv#3]
Hi,

when I run 'cd ./smartie-sv-master/ && make', it prompts an error message like this:

make[1]: Entering directory `/p200/chenhua_group/AISHA/Software/smartie-sv-master/src/blasr'
make[1]: *** No targets specified and no makefile found.  Stop.
make[1]: Leaving directory `/p200/chenhua_group/AISHA/Software/smartie-sv-master/src/blasr'
make: *** [bin/blasr] Error 2

How can I fix this? Am I missing on something?

Best,

**Comment by zeeev** [comment]
Hi,

Have you tried a recursive clone?

--Zev

**Comment by AishaYousaf** [comment]
Yeah. I first tried recursive clone which didn't work. So, I simply downloaded smartie, unzipped it and tried to make it. Are there any prerequisites to download along with smartie like blasr or htslib etc.?

Thanks!

**Comment by awaisfarooq724** [comment]
> Hi,
> 
> when I run 'cd ./smartie-sv-master/ && make', it prompts an error message like this:
> 
> make[1]: Entering directory `/p200/chenhua_group/AISHA/Software/smartie-sv-master/src/blasr' make[1]: *** No targets specified and no makefile found. Stop. make[1]: Leaving directory `/p200/chenhua_group/AISHA/Software/smartie-sv-master/src/blasr' make: *** [bin/blasr] Error 2
> 
> How can I fix this? Am I missing on something?
> 
> Best,

hi did you find any solution to it?
